bundle/config/artifact.go

```diff
@@ -1,20 +1,45 @@
 package config
 
-import "github.com/databricks/databricks-sdk-go/service/workspace"
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/databricks/databricks-sdk-go/service/compute"
+)
+
+type ArtifactType string
+
+const ArtifactPythonWheel ArtifactType = `whl`
 
 // Artifact defines a single local code artifact that can be
 // built/uploaded/referenced in the context of this bundle.
 type Artifact struct {
-	Notebook *NotebookArtifact `json:"notebook,omitempty"`
+	Type         ArtifactType `json:"type"`
+	Path         string       `json:"path"`
+	File         string       `json:"file"`
```

Comment:
From the outset it is not clear what path and file mean. Would be useful to have a comments for these fields. Looking at the code the path is the local path to the directory with `setup.py` and file is the (relative?) path to the wheel file itself?

Reply:
Yes, that's right
Would 
`Path` -> `PackagePath` (or `RootPath`) and `File` -> `Artifact` be more precise?

Reply:
I think they're fine but having field comments makes it better. If those are then surfaced through JSON schema then we can keep the field names as is.